src/python/deepgnn/graph_engine/snark/tests/convert_test.py

```diff
@@ -1431,6 +1431,93 @@ def test_edge_index_inverted_types(graph_with_inverse_edge_type_order):
         assert result[68:72] == struct.pack("f", -1)
 
 
+def graph_with_inverse_edge_dst_order_json(folder):
+    data = open(os.path.join(folder, "graph.json"), "w+")
+    graph = [
+        {
+            "node_id": 9,
+            "node_type": 0,
+            "node_weight": 1,
+            "edge": [
+                {
+                    "src_id": 9,
+                    "dst_id": 2,
+                    "edge_type": 1,
+                    "weight": 1.0,
+                },
+                {
+                    "src_id": 9,
+                    "dst_id": 1,
+                    "edge_type": 1,
+                    "weight": 0.5,
+                },
+            ],
+        },
+    ]
+    for el in graph:
+        json.dump(el, data)
+        data.write("\n")
+    data.flush()
+    data.close()
+
+    return data.name
+
+
+def graph_with_inverse_edge_dst_order_tsv(folder):
+    data = open(os.path.join(folder, "graph.tsv"), "w+")
+    data.write("9\t0\t1\tf:0 1;f:-0.01 -0.02\t2,1,1.0|1,1,0.5\n")
+    data.flush()
+    data.close()
+    return data.name
+
+
+@pytest.fixture(scope="module")
+def graph_with_inverse_edge_dst_order(request):
+    workdir = tempfile.TemporaryDirectory()
+    if request.param == JsonDecoder:
+        data_name = graph_with_inverse_edge_dst_order_json(workdir.name)
+    elif request.param == TsvDecoder:
+        data_name = graph_with_inverse_edge_dst_order_tsv(workdir.name)
+    else:
```

Comment:
Need for edge list as well, can just use json to edgelist converter already in file.

Reply:
EdgeList produces edges, I don't think we can sort it and test..